### ISSUE TITLE
[UX] CRM/ Leads

### ISSUE THREAD

**Issue by blaggacao** [odoo/odoo#15685]
This is an anecdotal report of CRM usability pains:

**As an Expert Salesperson,**

I need to have all relevant information readily available when working on a lead (eg. having someone from the client on the phone), so I would appreciate the following possibilities:

1. If I could update contact details (of company AND contact person) on the related res.partner(s) directly from the opportunity, if a partner has been set _(eg. the secretary is giving me the PBX extension of the main contact, so I would need to update that information on the fly)_
2. If I could have a wizard for correctly (!) creating a partner (create a partner AND it's contact person) also when we are in opportunity stage. _(The opportunity was created but not going through the lead stage)_
3. If I could have a wizard which would fix a wrongly created partner (so that the opportunity's contact information is transcribed and assigned to the partner). _(A simple "email-partner" was created by a welcome mail send to the lead, but as information completes, it needs to be properly reassigned to a newly created client partner)_
4. If I could have a new page which would be displaying the other contact persons of a company, at minimum with name, phone, mobile, email, function and with a little widget to add them as followers with one single click right from there. Updating information on a popup is ok for them (we are already in a later stage and have more settled and confident information about the partner). _(I want to be able to show the client that I really know very well it's buying center by dropping some relevant names in the conversation)_
5. If I get someone new on the phone, I want to immediately register the contact details and function for the record (Create-Button on page suggested in 4.) _(Client tells me on the phone to schedule a meeting together with colleagues X, Y and Z, so I want to schedule that meeting right away registering X, Y and Z with function and email on the fly and BEFORE hanging up in order to confirm right away the arrival of the email)_

@fpodoo imho, this is a topic you probably intimately care about, and I would not know whom to ping otherwise...
@Yenthe666 Do you know who else would be on this?

**Comment by Yenthe666** [comment]
Hmm I believe you've got the right guy, Fabien has to make the final decision probably.
Perhaps @mart-e can give feedback here too.

**Comment by mart-e** [comment]
Hello,

Thank you for your feedback. I have communicated your feedbacks to our UX team for the future versions. We tends to avoid the kind of wizard of point 2 and 3 so it unlikely we will do it.

We have already planned to make some improvements in the CRM, especially to make it easier the work of saleperson calling leads.
I think some changes like point 1 can already be easily done with small customization.

Closing as not a bug on Odoo
